### ISSUE TITLE
Regular merging into Master

### ISSUE THREAD

**Issue by liamhebert** [AndrewWay/cubesat#50]
Be careful of monolithic branches, the rpi branch in particular seems massive. A description of what each branch is would be very useful if they have a purpose outside of individual features. This could be tied to #49 where it could be nice to have a board and have pull requests tied to the board

**Comment by amellalalex** [comment]
Yes I am definitely bad for those haha. It is a type of snowball issue where by the time you realize the branch is gigantic you're halfway through implementing a new feature... The light at the end of the tunnel will show up once a prototype for the dock is complete in the `dock` branch and merged -> `develop` and then merged -> `master`. Good habits should pick up from a 'fresh' point. 

The `rpi` branch is particular in that I wanted to quickly run a demo on an RPI but there were a few adjustments to make for the library versions, but used the still-in-development `dock` branch. Hence the massive branch. But AFAIK it will be deleted once the demo has been documented or evolves into something more elaborate.
